src/components/SearchFiltersPanel/SearchFiltersPanel.js

```diff
@@ -35,12 +35,10 @@ import omit from 'lodash/omit';
 
 import routeConfiguration from '../../routeConfiguration';
 import { createResourceLocatorString } from '../../util/routes';
+import { propTypes } from '../../util/types';
 import { SelectSingleFilter, SelectMultipleFilter, InlineTextButton } from '../../components';
 import css from './SearchFiltersPanel.css';
 
-// Dropdown container can have a positional offset (in pixels)
-const FILTER_DROPDOWN_OFFSET = -14;
```

Comment:
Certificate filter was the first filter in the layout. Is this same offset needed for the current first filter?
(Just a guess, I don't remember why the offset was -14px - i.e. was there some responsive breakpoint that needed it or was it about something else.)

Reply:
Summary from elsewhere: should be fine, no changes needed.